woocommerce/woocommerce
# Add Admin Billing & Shipping order details visibility hooks

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #52006.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create an order using the WooCommerce > Orders admin page.
2. In your theme's `functions.php` file or custom plugin, test the new hooks using:
```php
add_filter( 'woocommerce_admin_order_data_display_billing_details', function( $is_visible ) {
	if ( ! current_user_can( 'administrator' ) ) {
		return false;
	}
} );
```
3. Log into your WP website using a non-admin account/role for e.g. Subscriber
4. Admin Billing order details should now be invisible based on filter return value:

![screenshot-2](https://github.com/user-attachments/assets/e2c90041-2769-4164-b1eb-970d44af33c8)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance



-   [] Major

#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
